eukcc/refine.py

```diff
@@ -175,16 +189,23 @@ def refine(state):
 
     smallbins = []
     for i, b in enumerate(bins):
+        logging.info(f'bin {b.state["fasta"]}')
         if b.state["quality"] is None:
             smallbins.append(i)
-            continue
-        elif b.state["quality"]["completeness"] < 50:
-            smallbins.append(i)
-            write_table(b.state, result_table)
+            logging.debug(f'Bin quality is None')
+            if len(b.state["marker_genes"]) == 0:
+                write_no_marker_genes_output(b.state, result_no_marker_set_bins)
             continue
         else:
-            write_table(b.state, result_table)
-
+            if b.state["marker_set"]["quality"] != "good":
+                logging.debug("Quality is not good")
```

Comment:
Is there a definition of "good" somwhere?

Reply:
kind of... https://github.com/EBI-Metagenomics/EukCC/blob/698c32dc725b64afed8dcbf0226b4e757e3edfa2/eukcc/treehandler.py#L23

Reply:
I would add that the README